lnschema_core/models.py

```diff
@@ -1740,7 +1748,7 @@ class Meta(Record.Meta, TracksRun.Meta, TracksUpdates.Meta):
     """A universal id (hash of the set of feature values)."""
     name: str = CharField(max_length=150, null=True, default=None)
```

Comment:
@sunnyosun, why did you not remove `default=None`?

Reply:
It seems all instances of CharField still have it.

We should have all our schema modules be consistent.